src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -1898,6 +1898,33 @@
     <Environment Issuer="https://www.webex.com/" ConfigurationEndpoint="https://webexapis.com/v1/.well-known/openid-configuration" />
   </Provider>
 
+  <!--
+                                                ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+                                                ██ ███ ██ ▄▄▄█▄ ▄██ ▄▄▀██ ▄▄▄ ██
+                                                ██ █ █ ██ ▄▄▄██ ███ ▄▄▀██ ███ ██
+                                                ██▄▀▄▀▄██ ▀▀▀█▀ ▀██ ▀▀ ██ ▀▀▀ ██
+                                                ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+  -->
+
+  <Provider Name="Weibo" Id="3523b2ce-c595-4f7b-9dda-9ce984fbf02d"
+            Documentation="https://open.weibo.com/wiki/%E6%8E%88%E6%9D%83%E6%9C%BA%E5%88%B6%E8%AF%B4%E6%98%8E">
+    <Environment Issuer="https://www.weibo.com/">
+      <Configuration AuthorizationEndpoint="https://api.weibo.com/oauth2/authorize"
+                     RevocationEndpoint="https://api.weibo.com/oauth2/revokeoauth2"
+                     TokenEndpoint="https://api.weibo.com/oauth2/access_token"
+                     UserinfoEndpoint="https://api.weibo.com/2/users/show.json" />
+    </Environment>
+
+    <Setting PropertyName="Display" ParameterName="display" Type="String" Required="false"
+             Description="The value used as the 'display' parameter (can be set to 'default', 'mobile', 'wap', 'client', or 'apponweibo' to adjust the authorization page display style)" />
+
+    <Setting PropertyName="ForceLogin" ParameterName="forceLogin" Type="String" Required="false"
```

Comment:
Hum, representing a boolean as a string works, but we should probably update the generator and the XML schema to support boolean settings. Do you want to give it a try? 😃

Reply:
Sounds good!

Reply:
We are resolving the same issue we should have resolved at Huawei Question 4 😄

Reply:
Haha yeah! (tho' it wasn't necessary for Huawei since we didn't use the userinfo endpoint 😄)